web-app/pages/_app.tsx

```diff
@@ -1,8 +1,30 @@
+/* eslint-disable react/jsx-props-no-spreading */
 import '../styles/globals.scss';
 import type { AppProps } from 'next/app';
+import Script from 'next/script';
+
+const googleAnalyticsId = process.env.NEXT_PUBLIC_GOOGLE_ANALYTICS_ID;
 
 function MyApp({ Component, pageProps }: AppProps) {
-  // eslint-disable-next-line react/jsx-props-no-spreading
-  return <Component {...pageProps} />;
+  return (
+    <>
+      <Script strategy="lazyOnload" src={`https://www.googletagmanager.com/gtag/js?id=${googleAnalyticsId}`} />
+
+      <Script strategy="lazyOnload">
+        {`
+        window.dataLayer = window.dataLayer || [];
+        function gtag(){dataLayer.push(arguments);}
+        gtag('js', new Date());
+
+        gtag('config', ${googleAnalyticsId});
+        `}
+      </Script>
+      <Component {...pageProps} />
+    </>
+  );
 }
 export default MyApp;
+
+// Set Up Google Analytics With Next.JS (Using Next.JS Script Component)
+// https://youtu.be/QAdtc7VWuNE
+// https://github.com/jarrodwatts/portfolio/blob/master/src/pages/_app.tsx#L23-L37
```

Comment:
@ozanisgor Looks good. Can you please move these 3 lines of comments to right above `const googleAnalyticsId`?

Reply:
Actually it's probably easy for me to make that edit right here online. I'll try.

Reply:
@ryancwalsh Thank you 🙏